pkg/operator/ceph/cluster/controller.go

```diff
@@ -263,6 +263,11 @@ func (c *ClusterController) onAdd(obj interface{}) {
 		return
 	}
 
+	if hasCleanupPolicy(clusterObj) {
```

Comment:
Why do we need this check in case with `onAdd()`?  Are we preventing the users from starting a new cluster with this clean up policy `on` ?

Reply:
If the operator is restarted, we will actually get the `onAdd()` event for existing clusters